mods/persistence/game/machinery/cryopod.dm

```diff
@@ -125,7 +125,12 @@
 		H.home_spawn = src
 		var/datum/mind/occupant_mind = occupant.mind
 		if(occupant_mind)
-			SSpersistence.AddToLimbo(occupant_mind, occupant_mind.unique_id, LIMBO_MIND, occupant_mind.key, occupant_mind.current.real_name, TRUE)
+			var/success = SSpersistence.AddToLimbo(occupant_mind, occupant_mind.unique_id, LIMBO_MIND, occupant_mind.key, occupant_mind.current.real_name, TRUE)
+			if(!success)
+				to_chat(occupant, SPAN_WARNING("Something has gone wrong while deserializing your character. Contact an admin!"))
```

Comment:
You probably meant serializing? Also, its highly possible that at this point the player has disconnected and won't know their character won't save. So, maybe warn the staff and put them somewhere safe?